storage-node/src/commands/server.ts

```diff
@@ -91,8 +91,8 @@ export default class Server extends ApiCommandBase {
     queryNodeEndpoint: flags.string({
```

Comment:
Following https://github.com/Joystream/joystream/pull/5001/files#r1443274494 I would recommend to update the name here as well. This would also apply to other command that use the same flag

Reply:
Addressed in [e2ce216](https://github.com/Joystream/joystream/pull/5001/commits/e2ce21622463a09f62c639ab6eee332d0678ecfd)